src/solidity_verifier/codegen.rs

```diff
@@ -594,6 +594,20 @@ pub fn solidity_aux_gen<E: MultiMillerLoop>(
     proofs: Vec<u8>,
     aux_file: &Path,
 ) {
+    let div_res = solidity_aux_genv2(params, vkey, instances, proofs, true);
+    let mut fd = std::fs::File::create(&aux_file).unwrap();
+    div_res
+        .iter()
+        .for_each(|res| res.write(&mut fd).unwrap());
+}
+
+pub fn solidity_aux_genv2<E: MultiMillerLoop>(
```

Comment:
please rename to `solidity_aux_gen_data`

Reply:
done